### PR TITLE
Add "packages.default" attribute

### PR DESCRIPTION
**Pull request by matthiasbeyer**

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
bors d=TheNeikos

**Comment by matthiasbeyer** [comment]
@TheNeikos Please have a look, I am not as experienced with flakes as you.

File: flake.nix [hunk 1]
```diff
@@ -68,6 +68,7 @@
           drv = changelog;
         };
         apps.default = apps.changelog;
+        packages.default = changelog;
```
Comment (TheNeikos) [inline flake.nix:71]: I would create a `packages.changelog = changelog` and then point to that as a default, similar to the devshell/apps.

**Comment by TheNeikos** [comment]
Yup, lgtm!

**Comment by matthiasbeyer** [comment]
bors merge

**Pull request merged**